internal/kubernetes/operator/config_exporter.go

```diff
@@ -160,9 +160,20 @@ func (e *ConfigExporter) Run() (string, error) {
 }
 
 func (e *ConfigExporter) exportProject() ([]runtime.Object, string, error) {
+	atlasProject, err := e.dataProvider.Project(e.projectID)
+	if err != nil {
+		return nil, "", err
+	}
+	if e.orgID != "" && e.orgID != atlasProject.OrgId {
+		return nil, "", fmt.Errorf("%q is not the owner of project %s, org id %s is",
+			e.orgID, atlasProject.GetId(), atlasProject.OrgId)
```

Comment:
```suggestion
		return nil, "", fmt.Errorf(`the project %s (%s) is not part of the organization %q.
Please confirm the arguments provided to the command or you are using the correct profile.",
			atlasProject.GetName(), atlasProject.GetId(), e.orgID)
```

Reply:
yes, this error message is much better, thanks!